### PR TITLE
(modified)button props added "textSizeIOS" for setting button size in…

### PR DESCRIPTION
**Pull request by gichang-yang**
Thanks for submitting a PR! Please read these instructions carefully:

- [x] Explain the **motivation** for making this change.
- [x] Provide a **test plan** demonstrating that the code is solid.
- [x] Match the **code formatting** of the rest of the codebase.
- [x] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)
I want to use 'react-native' more comfortable.

What existing problem does the pull request solve?
---------------------------------------------------------------------
->in this version, 'button' component couldn't change text size in 'ios'. so, I added button's props 
"textSizeIOS" for can change button's text size(May be it's similar 'change button size' in ios.)
-------------------------------------------------------------

 .  

## Test Plan (required)

A good test plan has the exact commands you ran and their output, provides screenshots or videos if the pull request changes UI or updates the website. See [What is a Test Plan?][1] to learn more.  

If you have added code that should be tested, add tests.
-----------------------------------------------------------------
Ans:
# **->testing video is here. text size is 10,20 and 30.
[IMG_1944.MOV.zip](https://github.com/facebook/react-native/files/981039/IMG_1944.MOV.zip)**
--------------------------------------------------------------------

## Next Steps

Sign the [CLA][2], if you haven't already.

Small pull requests are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

Make sure all **tests pass** on both [Travis][3] and [Circle CI][4]. PRs that break tests are unlikely to be merged.

For more info, see the ["Pull Requests"][5] section of our "Contributing" guidelines.

[1]: https://medium.com/@martinkonicek/what-is-a-test-plan-8bfc840ec171#.y9lcuqqi9
[2]: https://code.facebook.com/cla
[3]: https://travis-ci.org/facebook/react-native
[4]: http://circleci.com/gh/facebook/react-native
[5]: https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests

### CODE REVIEW

**Comment by hramos** [comment]
Hey, thanks for sending a pull request!\n\nIt looks like your description is missing some necessary information. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/PULL_REQUEST_TEMPLATE.md)? This is necessary for people to be able to understand and review the change you are proposing.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +1,31 @@
 Thanks for submitting a PR! Please read these instructions carefully:
 
-- [ ] Explain the **motivation** for making this change.
-- [ ] Provide a **test plan** demonstrating that the code is solid.
-- [ ] Match the **code formatting** of the rest of the codebase.
-- [ ] Target the `master` branch, NOT a "stable" branch.
+- [x] Explain the **motivation** for making this change.
```
Comment (hramos) [inline .github/PULL_REQUEST_TEMPLATE.md]: You don't need to update the PULL_REQUEST_TEMPLATE file itself.

File: Libraries/Components/Button.js [hunk 2]
```diff
@@ -57,33 +56,38 @@ class Button extends React.Component {
...
+    onPress: React.PropTypes.func.isRequired,
     /**
      * Used to locate this view in end-to-end tests.
      */
-    testID: PropTypes.string,
+    testID: React.PropTypes.string,
+      /**
+       * Size to change button's textSize (only in IOS)
+       */
+      textSizeIos: React.PropTypes.string,
```
Comment (hramos) [inline Libraries/Components/Button.js]: The indentation here does not match the rest of the file.

File: Libraries/Components/Button.js [hunk 3]
```diff
@@ -94,6 +98,7 @@ class Button extends React.Component {
       title,
       disabled,
       testID,
+        textSizeIos,
```
Comment (hramos) [inline Libraries/Components/Button.js]: Same here.

File: Libraries/Components/Button.js [hunk 4]
```diff
@@ -14,7 +14,6 @@
 const ColorPropType = require('ColorPropType');
 const Platform = require('Platform');
 const React = require('React');
-const PropTypes = require('prop-types');
```
Comment (hramos) [inline Libraries/Components/Button.js]: Can we make this change in a separate PR?

File: Libraries/Components/Button.js [hunk 5]
```diff
@@ -57,33 +56,38 @@ class Button extends React.Component {
...
     testID?: ?string,
+    textSizeIos?: ?string,
   };
 
   static propTypes = {
     /**
      * Text to display inside the button
      */
-    title: PropTypes.string.isRequired,
+    title: React.PropTypes.string.isRequired,
```
Comment (hramos) [inline Libraries/Components/Button.js]: Let's focus on a single change at a time. Handle proptypes in a separate PR.

File: Libraries/Components/Button.js (hunk 4 above)
Comment (gichang-yang) [reply Libraries/Components/Button.js]: That means, Remaining(const PropTypes = require...) is better?
